### PR TITLE
feature_release setup:267 set a mail password via doil.conf

### PR DESCRIPTION
**Pull request by daniwe4**
https://github.com/conceptsandtraining/doil/issues/267

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -301,6 +301,13 @@ login data:
 * User: www-data
 * Password: ilias
 
+You can change the password in the **doil** config file `setup/conf/doil.conf`.
+Before installing **doil** change the default password to your password.
+If doil is already installed you can change the password by `doil mail:change-password <password>`.
+Please ensure you also update the password in your actual installed
+**doil** config `/etc/doil/doil.conf`, this would protect you to set the passwort again after a doil update.
```
Comment (tbongers-cat) [inline README.md:308]: _"... this would protect you from having to set the password again..."_ sounds a bit better to me

**Review by tbongers-cat — APPROVED** [review]
Just a little phrasing in the Readme, otherwise this looks good

File: README.md (hunk 1 above)
Comment (tbongers-cat) [reply README.md:308]: Alternatively: _"..., so you don't have to set the password again ..."_

Comment (daniwe4) [reply README.md]: Done!

**Review by tbongers-cat — APPROVED** [review]
lgtm

**Pull request merged**